OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -161,22 +161,22 @@ public override bool Tick(Actor self)
 			if (insideMaxRange && !insideMinRange)
 				return true;
 
-			var isSlider = aircraft.Info.FlightDynamics.HasFlag(FlightDynamic.Slide);
+			var isSlider = aircraft.Info.CanSlide;
```

Comment:
Probably here too, but this is used in several places so could go either way.

Reply:
I've left this as-is for now.